stream.go

```diff
@@ -155,6 +155,15 @@ func (cs *clientStream) Trailer() metadata.MD {
 }
 
 func (cs *clientStream) SendMsg(m interface{}) (err error) {
+	if EnableTracing {
+		cs.traceInfo.tr.LazyLog(payload{m}, true)
```

Comment:
I am not convinced that we should log all outbound payload (and inbound payload) for streaming rpc. There could be huge.